test/integration_test.py

```diff
@@ -1565,12 +1570,35 @@ def assert_manifest(expected_bundles):
         # With authorized credentials, all bundles included in the filters
         # should be represented in the manifest
         with self._service_account_credentials:
-            assert_manifest({public_bundle, *managed_access_bundles})
+            test_compact_manifest(all_bundles)
 
         # Without credentials, only the public bundle should be represented
-        assert_manifest({public_bundle})
+        test_compact_manifest({public_bundle})
 
-        if ManifestFormat.curl in self.metadata_plugin(catalog).manifest_formats:
+        def dump_verbatim_manifest(bundles: Iterable[str]) -> set[str]:
+            manifest_url = furl(url=endpoint, path='/manifest/files', args={
+                'catalog': catalog,
+                'format': ManifestFormat.verbatim_jsonl.value,
+                'filters': json.dumps({special_fields.bundle_uuid: {'is': list(bundles)}})
+            })
+            manifest_lines = BytesIO(self._get_url_content(PUT, manifest_url)).readlines()
+            manifest_content = {
+                freeze(json.loads(replica))
+                for replica in manifest_lines
+            }
+            self.assertEqual(len(manifest_lines), len(manifest_content))
+            return manifest_content
+
+        for format in ManifestFormat.verbatim_jsonl, ManifestFormat.verbatim_pfb:
```

Comment:
`format` isn't used anywhere except for `subTest()` so this code needlessly is doing the same thing twice. It looks like `dump_verbatim_manifest` needs to be extended to handle PFB manifests.